crawl-ref/source/webserver/webtiles/config.py

```diff
@@ -166,8 +165,7 @@ def load_game_data():
     # TODO: should the `load_games` module be refactored into config?
     global games
     games = get('games', collections.OrderedDict())
-    if get('use_game_yaml', False):
```

Comment:
This check or the equiv really needs to be there for dgamelaunch-config setups; they use a `cp -r` from the repo directory to install the webtiles server, and those files are untemplated, so the default behavior installs broken yml files that prevent webtiles startup on dgl-config if this conditional isn't there. Something could be changed in dgl-config of course but since those scripts are never updated automatically I'd rather keep this centralized. See bec1141c92d9.

Reply:
Hm. So the default value for it (as defined further up in this file) is True. So the DGL servers have a config.py template that explicitly includes `use_game_yaml = False`? I'll keep the exact config name then for simplicity.

Reply:
hm, I originally defaulted it to `False` (with `True` in the repo config) but then c07941e8bc853 changed it to `True` (which would require an explicit setting); I'm not really sure if that was intentional on my part, but all webtiles servers have probably restarted since then, so I suppose admins must have fixed it. Probably `True` is the "right" default.